cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -46,6 +47,23 @@
 
     public static final String CRUISE_CONTROL_GOALS = String.join(",", CRUISE_CONTROL_GOALS_LIST);
 
+    /**
+     * A list of case insensitive goals that Cruise Control will use as hard goals that must all be met for an optimization
+     * proppsal to be valid.
+     */
+    protected static final List<String> CRUISE_CONTROL_HARD_GOALS_LIST = Collections.unmodifiableList(
+            Arrays.asList(
+                    CruiseControl.RACK_AWARENESS_GOAL,
+                    CruiseControl.REPLICA_CAPACITY_GOAL,
+                    CruiseControl.DISK_CAPACITY_GOAL,
+                    CruiseControl.NETWORK_INBOUND_CAPACITY_GOAL,
+                    CruiseControl.NETWORK_OUTBOUND_CAPACITY_GOAL
+                    //CruiseControl.CPU_CAPACITY_GOAL
```

Comment:
Should we maybe add the same explnatory comment as we have above?

Reply:
Will do.